tasks/main.yml

```diff
@@ -15,3 +15,11 @@
     version: "{{ omero_omego_version }}"
     virtualenv: "{{ omero_common_basedir }}/omego"
     virtualenv_site_packages: true
+
+- name: omego | install future
+  become: true
+  pip:
+    name: future
+    state: present
```

Comment:
I assume with https://github.com/ome/omego/pull/122, `future` should be installed as part of the `pip` module? Is it worth bumping the version of `omero_omego_version` and/or making a full (non-dev) release of `omego` at this stage?

Reply:
:+1: I assume we go for a full release at this point, eh?

Reply:
If `future` needs to be manually installed it sounds like a bug in `omego`